server/modules/nodeDef/manager/nodeDefManager.js

```diff
@@ -53,6 +55,49 @@ const updateNodeDefProps = async (user, surveyId, nodeDefUuid, props, propsAdvan
     return nodeDef
   })
 
+const deleteNodeDefValidationMessagesLabels = async (surveyId, deletedLanguages, client = db) => {
+  await client.tx(async tx => {
```

Comment:
what about calling this method ```deleteValidationMessageLabels```? ( I know, we'll eventually remove the nodeDef prefix to other methods as well)

Reply:
Now it's consistent with the other names, we will change it in the future.